lookout/style/format/descriptions.py

```diff
@@ -1,17 +1,39 @@
 """Features and rules description utils."""
 from collections import defaultdict
+from copy import copy
 from functools import singledispatch
 from math import ceil, floor
 from typing import List, Tuple
 
 from numpy import flatnonzero, floating, ndarray
 
 from lookout.style.format.feature_extractor import FeatureExtractor, FEATURES_MAX, FEATURES_MIN
-from lookout.style.format.feature_utils import CLASSES
+from lookout.style.format.feature_utils import (
+    CLASS_INDEX, CLASSES, CLS_DOUBLE_QUOTE, CLS_NEWLINE, CLS_NOOP, CLS_SINGLE_QUOTE, CLS_SPACE,
+    CLS_SPACE_DEC, CLS_SPACE_INC, CLS_TAB, CLS_TAB_DEC, CLS_TAB_INC)
 from lookout.style.format.features import BagFeature, CategoricalFeature, OrdinalFeature
 from lookout.style.format.rules import Rule
 
 
+_CLASS_REPRESENTATIONS_MAPPING = {
+    CLS_DOUBLE_QUOTE: '"',
+    CLS_NEWLINE: "⏎",
+    CLS_NOOP: "∅",
+    CLS_SINGLE_QUOTE: "'",
+    CLS_SPACE: "␣",
+    CLS_SPACE_DEC: "␣⁻",
+    CLS_SPACE_INC: "␣⁺",
+    CLS_TAB: "⇥",
+    CLS_TAB_DEC: "⇥⁻",
+    CLS_TAB_INC: "⇥⁺",
+}
+CLASS_REPRESENTATIONS = [_CLASS_REPRESENTATIONS_MAPPING[cls] for cls in CLASSES]
```

Comment:
We should `del _CLASS_REPRESENTATIONS_MAPPING`, otherwise there will be abusers.